names/names_test.go

```diff
@@ -94,7 +95,7 @@ func TestNames(t *testing.T) {
 		{"RepositoryUriTest", "RepositoryURITest", "repositoryURITest", "repository_uri_test", "repositoryuritest"},
 		{"RequestedAmiVersion", "RequestedAMIVersion", "requestedAMIVersion", "requested_ami_version", "requestedamiversion"},
 		{"SaslScram512Auth", "SASLSCRAM512Auth", "saslSCRAM512Auth", "sasl_scram_512_auth", "saslscram512auth"},
-		{"Secret", "Secret", "secret", "secret", "secret"},
+		{"Secret", "SECRET", "secret", "secret", "secret"},
```

Comment:
this should be: `{"Secret", "Secret", "secret", "secret", "secret"}`

Reply:
fixed [- remove Secret from initialisms](https://github.com/aws-controllers-k8s/pkg/pull/21/commits/b2030dde27058a276fc32dd5c5f8db7cff3e36c7)